packages/twenty-front/src/modules/settings/data-model/components/SettingsObjectFieldSelectForm.tsx

```diff
@@ -72,6 +74,45 @@ export const SettingsObjectFieldSelectForm = ({
     onChange(nextOptions);
   };
 
+  const handleUniqueDefaultValueChange = (
+    index: number,
+    option: SettingsObjectFieldSelectFormOption,
+    nextOption: SettingsObjectFieldSelectFormOption,
+  ) => {
+    const hasDefaultOptionChanged = option.isDefault !== nextOption.isDefault;
+    const nextOptions = hasDefaultOptionChanged
+      ? values.map((value) => ({
+          ...value,
+          isDefault: false,
+        }))
+      : [...values];
+
+    nextOptions.splice(index, 1, nextOption);
+
+    onChange(nextOptions);
+  };
+
+  const handleDefaultValueChange = (
```

Comment:
the distinction between Unique and not unique is not clear to me. I would prefer something more explicit: handleMultiSelectDefaultValueChange vs handleSelectDefaultValueChange

Reply:
ok